photon-targeting/build.gradle

```diff
@@ -191,6 +189,27 @@ model {
 
 apply from: "${rootDir}/shared/javacpp/publish.gradle"
 
+// quickly hack our raw jar into publishing
+def rawjavaJar = tasks.register("rawjavaJar", Jar) {
```

Comment:
purpose?

Reply:
https://discord.com/channels/176186766946992128/368993897495527424/1306457242979139584

Reply:
TLDR:
photontargeting-java includes .so's
But we deploy them to the robot via jniDependencies: photontargeting-cpp
So we need to strip them from the JAR we ship to the robot